Examples/Physics_applications/capacitive_discharge/analysis.py

```diff
@@ -15,5 +15,5 @@
 
 my_check = checksumAPI.evaluate_checksum(
     'background_mcc', 'Python_background_mcc_plt00050',
-    do_particles=True, rtol=2.5e-3
+    do_particles=True, rtol=3.7e-3
```

Comment:
@roelof-groenewald I changed this temporarily to 3.7e-3 and that was the relative difference for particle_momentum_z for electrons. All other quantities in the checksum are within this tolerance. Was 2.5e-3 chosen specifically or was this driven by the maximum difference in the checksum? 
This PR changes when time get updates, and as a result, background_mcc benchmarks are updated since the potential hi uses the updated time

Reply:
That sounds fine. The 2.5e-3 tolerance was not intelligently chosen, but by the needed tolerance for the test to pass so increasing it to 3.7e-3 is fine.

Reply:
Thanks a lot!